docs/user/library/pinnacle.rst

```diff
@@ -0,0 +1,24 @@
+####################
+Pinnacle Export Tool
+####################
+
+*******
+Summary
+*******
+
+.. automodule:: pymedphys_pinnacle.export
+    :no-members:
+
+
+***
+API
+***
+
+.. autoclass:: pymedphys_pinnacle.export.PinnacleExport
```

Comment:
From here reference `pymedphys.PinnacleExport` instead of `pymedphys_pinnacle`. The sub packages should be transparent to users, never a part of the public API.

Reply:
@pchlap this is an easy fix, so I'll do this one, then merge it.